transform/mattermost-analytics/models/staging/mattermost2/stg_mattermost2__tracks.sql

```diff
@@ -0,0 +1,18 @@
+
+WITH tracks AS (
+    SELECT
+        {{ dbt_utils.star(ref('base_mattermost2__tracks')) }}
+    FROM
+        {{ ref ('base_mattermost2__tracks') }}
+)
+SELECT
+     id               AS event_id
+     , event          AS event_table
+     , event_text     AS event_name
+     , category       AS category
+     , type           AS event_type
+     , user_id        AS server_id
+     , user_actual_id AS user_id
```

Comment:
couldn't find user_actual_id for raw.mattermost2.tracks though.

Reply:
The `user_actual_id` for mattermost2 is in `mattermost2.event` table

Reply:
Yes, my bad, `get_event_relations()` gets them from all `rudderstack_tables`.

Reply:
@catalintomai exactly, it's the same idea as with rudderstack data. `user_actual_id` is a custom column, thus never added in `tracks` table. The workaround is to create a "synthetic" tracks view that unions all event tables. Note that the synthetic tracks table is a view, and not a table. The view can be always updated to include new tables, without a need to do a full refresh.